src/metadataAccessor.ts

```diff
@@ -36,6 +36,6 @@ export class MetadataAccessor<T extends Object> {
             return null;
         }
 
-        throw new Error(`Array found for '${targetPropertyName}' field with no type declared. Use @anyArray or @typedArray decorators.`);
+        throw new Error(`Array found for '${targetPropertyName.toString()}' field with no type declared. Use @anyArray or @typedArray decorators.`);
```

Comment:
i thought it works like this by default?

Reply:
I also thought, but it throws an error. You can try it by yourself, just print in browser console: `${Symbol()}`.